.grit/dealias_trulens_eval.grit

```diff
@@ -506,8 +506,8 @@ any {
   `from trulens_eval.streamlit import FeedbackDisplay as $alias` => `from trulens.dashboard.streamlit import FeedbackDisplay as $alias`,
   `from trulens_eval.streamlit import trulens_feedback` => `from trulens.dashboard.streamlit import trulens_feedback`,
   `from trulens_eval.streamlit import trulens_feedback as $alias` => `from trulens.dashboard.streamlit import trulens_feedback as $alias`,
-  `from trulens_eval.streamlit import trulens_leaderboard` => `from trulens.dashboard.streamlit import trulens_leaderboard`,
-  `from trulens_eval.streamlit import trulens_leaderboard as $alias` => `from trulens.dashboard.streamlit import trulens_leaderboard as $alias`,
+  `from trulens_eval.streamlit import trulens_leaderboard` => `from trulens.dashboard.leaderboard import render_leaderboard`,
```

Comment:
can we keep streamlit/trulens names for these components? module is named streamlit because the components are specific to streamlit dashboards, and trulens is used in the name to differentiate it when it's used in streamlit code

Reply:
@coreyhu I see you resolved this, but the naming is still changed. Can we keep the current names for these?